### DISCUSSION TITLE
LTMEAT FLOOD not working

### DISCUSSION THREAD

**Discussion by Blobby-Boi** [3kh0/ext-remover#974] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I added the recommended 1000 tabs and when I do it the tabs get unblocked but only for a minute or so
When I try more tabs the chromebook crashes

**Comment by AshtonDavies** [comment]
How many more tabs are you adding? You may be adding too many. Generate the folder somewhere between that, and make sure all other tabs are closed and unnecessary extensions are disabled.

A common cause of this issue is probably having an extension that uses too many resources, such as Grammarly.

**Reply by Blobby-Boi** [reply]
I found that the max number of tabs without crashing is 1150. That disables it for about 15 minutes but then it re-enables. I made sure to turn off every extension.

**Reply by AshtonDavies** [reply]
> I found that the max number of tabs without crashing is 1150. That disables it for about 15 minutes but then it re-enables. I made sure to turn off every extension.

I think the permanent freeze has been fixed at some point. You're probably better off using [Dextensify](https://chrose.pages.dev/detail/Dextensify), which sucks, but at least it's quicker to perform and may or may not last longer than 15mins.

**Reply by Blobby-Boi** [reply]
Dextensify has just never worked for me. It either crashes the Chromebook, lags it to the point where it’s unusable, or just doesn’t work. Is there any other alternative?

**Reply by AshtonDavies** [reply]
> Dextensify has just never worked for me. It either crashes the Chromebook, lags it to the point where it’s unusable, or just doesn’t work. Is there any other alternative?

It has recently started to work for me, and from testing, it even seems to work on other devices such as laptops with Windows.

If you give up on Dextensify, maybe Corkey or Crosh (if enabled by admin) would be a working alternative.

**Reply by Blobby-Boi** [reply]
I checked again, and it still crashes… My Chromebook recently was forced to update to chromeos 122, so maybe it got patched.

**Reply by AshtonDavies** [reply]
It actually does work on v122.

**Reply by v4pl** [reply]
@Blobby-Boi join our discord sever

**Reply by v4pl** [reply]
https://discord.gg/VQr7E7D7CV

**Reply by Blobby-Boi** [reply]
I joined

**Reply by Blobby-Boi** [reply]
LTMEAT Print worked now

**Reply by AshtonDavies** [reply]
> LTMEAT Print worked now

I doubt that would last longer but okay.

**Reply by Blobby-Boi** [reply]
It’s lasted for about 12 hours cuz I found a way to make it last forever

**Reply by AshtonDavies** [reply]
> It’s lasted for about 12 hours cuz I found a way to make it last forever

Print does not last forever. Just as long as the length of the page you use for it.

**Reply by Blobby-Boi** [reply]
It’s 150k pages lol

**Reply by xoidex** [reply]
what extension has 150k pages 😭

**Reply by AshtonDavies** [reply]
> what extension has 150k pages 😭

Lightspeed Filter Agent has a big fat main.js stored in it.

**Reply by Blobby-Boi** [reply]
I used lightspeed and made the margins low and everything

**Reply by Blobby-Boi** [reply]
Btw an update is that after 17 hours it did in fact stop working. But that’s so much better than the 15 minutes with the flood method

**Comment by S-PScripts** [comment, marked as answer]
skibidi

**Reply by Blobby-Boi** [reply]
what even is this 😭
